app/views/categories/show.html.haml

```diff
@@ -3,7 +3,7 @@
   .col-md-12
     = link_to group_categories_path(current_group), :class => "btn btn-default" do
       = icon 'arrow-left'
-    -if can_see?(:manage, @category)
+    -if can_see? :update, @category
```

Comment:
the action that you want show is edit, so you need to have update access for seeing it